build/lib/install_go_versions.sh

```diff
@@ -53,6 +54,9 @@ GOBIN=${GOPATH}/go1.17/bin go install github.com/google/go-licenses@v1.2.1
 build::common::use_go_version "1.18"
 GOBIN=${GOPATH}/go1.18/bin go install github.com/google/go-licenses@v1.2.1
 
+build::common::use_go_version "1.19"
+GOBIN=${GOPATH}/go1.19/bin go install github.com/google/go-licenses@v1.0.0
```

Comment:
Why are we downgrading go-licenses to v1.0.0 here?

Reply:
will fix it. it was part of the test with 1.19.